andela/bp-andela-media
# #161593689 Create an album

#### What does this PR do?
- Add functionality to create an album

#### Description of Task to be completed?
- Create album model
- Create a user group model
- Add create album controller action
- Write tests

#### How should this be manually tested?
- Open postman and go to `localhost:3000/albums`
- Add required fields and check the response

#### What are the relevant pivotal tracker stories?
- [#161593689](https://www.pivotaltracker.com/story/show/161593689)

#### Screenshots
- Creating album with an existing user group
<img width="1010" alt="screen shot 2018-12-17 at 15 34 39" src="https://user-images.githubusercontent.com/27849036/50087528-642ed080-0211-11e9-8abd-8b19a195d056.png">

- Creating an album and user group at the same time
<img width="1050" alt="screen shot 2018-12-17 at 15 32 16" src="https://user-images.githubusercontent.com/27849036/50087540-6ee96580-0211-11e9-98f6-897e8a2bf783.png">
